e2e/cypress/integration/ui/settings/03_team_add_users.spec.ts

```diff
@@ -0,0 +1,177 @@
+describe('team add users', () => {
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress test';
+  const nameForUser = cypressPrefix + ' user ' + now;
+  const usernameForUser = 'testing-user-' + now;
```

Comment:
we could interpolate these strings

Reply:
Is there an advantage to interpolating vs what I have?

Reply:
it's easier to read, imo, and it'd be consistent with the commands in the rest of the file (like block of route requests L66-72)